### PR TITLE
geo: rename To2DShapeType to To2D, introduce ShapeType2D method

### PR DESCRIPTION
**Pull request by otan**
This avoids the ShapeType().To2DShapeType() verbosity.

Release note: None

### CODE REVIEW

File: pkg/geo/geo.go [hunk 1]
```diff
@@ -305,6 +305,11 @@ func (g *Geometry) ShapeType() geopb.ShapeType {
 	return g.spatialObject.ShapeType
 }
 
+// ShapeType returns the 2D shape type of the Geometry.
```
Comment (andyyang890) [inline pkg/geo/geo.go:308]:
```suggestion
// ShapeType2D returns the 2D shape type of the Geometry.
```

File: pkg/geo/geo.go [hunk 2]
```diff
@@ -552,6 +557,11 @@ func (g *Geography) ShapeType() geopb.ShapeType {
 	return g.spatialObject.ShapeType
 }
 
+// ShapeType returns the 2D shape type of the Geography.
```
Comment (andyyang890) [inline pkg/geo/geo.go:560]:
```suggestion
// ShapeType2D returns the 2D shape type of the Geography.
```

File: pkg/geo/geo.go [hunk 3]
```diff
@@ -305,6 +305,11 @@ func (g *Geometry) ShapeType() geopb.ShapeType {
 	return g.spatialObject.ShapeType
 }
 
+// ShapeType returns the 2D shape type of the Geometry.
+func (g *Geometry) ShapeType2D() geopb.ShapeType {
+	return g.spatialObject.ShapeType.To2D()
```
Comment (andyyang890) [inline pkg/geo/geo.go:310]: Is there a reason you chose not to use `g.ShapeType().To2D()`?

File: pkg/geo/geo.go [hunk 4]
```diff
@@ -552,6 +557,11 @@ func (g *Geography) ShapeType() geopb.ShapeType {
 	return g.spatialObject.ShapeType
 }
 
+// ShapeType returns the 2D shape type of the Geography.
+func (g *Geography) ShapeType2D() geopb.ShapeType {
+	return g.spatialObject.ShapeType.To2D()
```
Comment (andyyang890) [inline pkg/geo/geo.go:562]: Same question as above

**Review by andyyang890 — APPROVED** [review]

**Review by otan — COMMENTED** [review]
thanks. but remember you really don't have to review these on a weekend!

File: pkg/geo/geo.go (hunk 3 above)
Comment (otan) [reply pkg/geo/geo.go]: Nothing in particular - just copy paste. Doesn't matter too much, heh.

**Comment by otan** [comment]
bors r=andyyang890

**Pull request merged**